Assets/MRTK/Core/Providers/BaseSpatialObserver.cs

```diff
@@ -40,6 +40,13 @@ public abstract class BaseSpatialObserver : BaseDataProvider<IMixedRealitySpatia
         [System.Obsolete("Call Service instead.")]
         protected IMixedRealitySpatialAwarenessSystem SpatialAwarenessSystem => Service;
 
+        protected GameObject observedObjectParent = null;
+
+        /// <summary>
+        /// The parent GameObject for all observed meshes to be placed under.
+        /// </summary>
+        public virtual GameObject ObservedObjectParent => observedObjectParent != null ? observedObjectParent : (observedObjectParent = Service?.CreateSpatialAwarenessObservationParent(Name));
```

Comment:
In order to prevent this from being a breaking change, would we want to add a new `public` property that points to the existing `protected virtual` one? That way any class that already overrides it would be fine without the visibility change, but anybody who needs access can get it.

Reply:
Discussed offline. This feels like a break we can contain.

Reply:
again, on further review, i like your proposal